### DISCUSSION TITLE
Login Issue

### DISCUSSION THREAD

**Discussion by GemengNub** [ppy/osu#21297] [Q&A] [answered]
### Type

Game behaviour

### Bug description

When I enter my password and login, it says forbidden.

Windows  10 Home 21H2

### Screenshots or videos

![image](https://user-images.githubusercontent.com/115711213/202671413-544fe72c-9455-4b87-93d8-613d513329f0.png)

### Version

2022.1117.0 (Nov 17 2022)

### Logs

[network.log](https://github.com/ppy/osu/files/10040162/network.log)
[performance.log](https://github.com/ppy/osu/files/10040163/performance.log)
[runtime.log](https://github.com/ppy/osu/files/10040164/runtime.log)
[updater.log](https://github.com/ppy/osu/files/10040165/updater.log)
[database.log](https://github.com/ppy/osu/files/10040166/database.log)
[input.log](https://github.com/ppy/osu/files/10040167/input.log)

**Comment by peppy** [comment, marked as answer]
Looks like your IP is blacklisted by cloudflare. You may be able to fix things by visiting https://osu.ppy.sh and completing cloudflare verfication.

**Reply by GemengNub** [reply]
when I go to osu! website, it automatically let's me pass through but doesn't let me login(on osu lazer), I figured to use a VPN and worked, thanks for the heads up!

**Reply by peppy** [reply]
Good to know. This generally means that cloudflare thinks your IP is a bad actor. I'd double check your PC (or another on your network) isn't compromised and acting as part of a botnet.

If you have a dynamic IP, it would pay to reboot your router or ask your ISP for a new one if you need to.
